### PR TITLE
feat: [tour] added single-banner block option for banner carousel

### PR DESCRIPTION
**Pull request by etomaello**
Added the single-banner block option for the banner-carousel block, it renders the carousel as a single banner without slide indicators, background watermarks and h2+h3 texts positioned in the lower end.

Test URLs:
- Before: https://main--realmadrid--hlxsites.hlx.page/sites/tour-bernabeu/colegios
- After: https://feat-banner-carousel-single_banner--realmadrid--hlxsites.hlx.page/sites/tour-bernabeu/colegios

Sharepoint url: https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7BB02562A4-02C4-4396-9FD8-AB29618356D8%7D&file=colegios.docx&action=default&mobileredirect=true&cid=8f602360-c837-4032-8891-3b0a7af848e6

### CODE REVIEW

File: blocks/banner-carousel/banner-carousel.css [hunk 1]
```diff
@@ -230,3 +230,43 @@ body main .section.banner-carousel-container {
...
   }
 }
+
+/*
+banner carousel - single banner option styling
+ */
+.banner-carousel.single-banner .carousel-slide h3 {
+  top: auto;
+  bottom: 105px;
+  left: 20px;
```
Comment (vdua) [inline blocks/banner-carousel/banner-carousel.css:240]: please use `inset-inline-start` to make it work in RTL

File: blocks/banner-carousel/banner-carousel.css [hunk 2]
```diff
@@ -230,3 +230,43 @@ body main .section.banner-carousel-container {
...
+  text-align: left;
```
Comment (vdua) [inline blocks/banner-carousel/banner-carousel.css:241]: `text-align : start` to make it work in RTL

File: blocks/banner-carousel/banner-carousel.css [hunk 3]
```diff
@@ -230,3 +230,43 @@ body main .section.banner-carousel-container {
...
+  font-size: var(--heading-font-size-l);
+  line-height: var(--heading-font-size-l);
+  font-weight: 500;
+}
+
+.banner-carousel.single-banner .carousel-slide h2 {
+  bottom: 35px;
+  left: 20px;
```
Comment (vdua) [inline blocks/banner-carousel/banner-carousel.css:249]: `inset-inline-start`

File: blocks/banner-carousel/banner-carousel.css (hunk 1 above)
Comment (etomaello) [reply blocks/banner-carousel/banner-carousel.css]: done!

File: blocks/banner-carousel/banner-carousel.css (hunk 2 above)
Comment (etomaello) [reply blocks/banner-carousel/banner-carousel.css]: done!

File: blocks/banner-carousel/banner-carousel.css (hunk 3 above)
Comment (etomaello) [reply blocks/banner-carousel/banner-carousel.css]: done!

File: blocks/banner-carousel/banner-carousel.css [hunk 4]
```diff
@@ -230,3 +230,43 @@ body main .section.banner-carousel-container {
...
+  top: auto;
+  text-align: left;
```
[the remark this answers is not in the document]
Comment (etomaello) [reply blocks/banner-carousel/banner-carousel.css]: done!

**Review by vdua — APPROVED** [review]

**Review by chicharr — APPROVED** [review]
LGTM

**Pull request merged**